ITEvents/Shared collection view elements/UIGradientImageView.swift

```diff
@@ -30,7 +30,7 @@ class UIGradientImageView: UIImageView {
         setGragientLayer(color: UIColor.white)
     }
     
-    //TODO do we need this code? Now we have only shown or not gradient
+    //TODO remove code when will remove DisplaySwitcher from FavoriteEventsView
```

Comment:
small miss print in the comment
//TODO remove code when **we** remove DisplaySwitcher from FavoriteEventsView